preup/kickstart/application.py

```diff
@@ -210,10 +210,28 @@ def comment_kickstart_issues(self):
                 kickstart_data[index] = "#" + row
         FileHelper.write_to_file(self.kick_start_name, 'wb', kickstart_data)
 
+    def _get_all_postinstall_scripts(self):
+        found_scripts = []
+        for (path, dirs, files) in os.walk(os.path.join(settings.result_dir, settings.kickstart_dir)):
+            for f in files:
+                file_name = os.path.join(path, f)
+                if os.path.exists(file_name) and FileHelper.check_file(file_name, u"x") is True:
+                    found_scripts.append(file_name)
+        return found_scripts
+
     def generate(self):
         if not self.collect_data():
             log_message("Important data are missing for the Kickstart generation.", level=logging.ERROR)
             return None
+        if not self._get_all_postinstall_scripts():
+            accept = ['y', 'yes']
+            log_message("You did not specify any postinstall scripts. "
+                        "Include them to %s " % os.path.join(settings.result_dir, settings.postinstall_dir))
+            choice = MessageHelper.get_message(message="Do you want to continue with kickstart generation "
+                                               "without postinstall scripts?",
+                                               prompt="(Y/n)")
+            if choice.lower() not in accept:
+                return None
```

Comment:
The *generate* method is called from the main method and when None is returned here, the main method continues in executing. Is it correct/expected behavior? Shouldn't the main return as soon as it receives None from here, with some log message ideally?